pkg/artifactory/datasource_artifactory_file_test.go

```diff
@@ -13,46 +16,148 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+//Provides a temporary path. The directory is automatically removed by Cleanup
+//when the test and all its subtests complete.
+func getTempDownloadPath(t *testing.T) string {
+	return fmt.Sprintf("%s/crash.zip", t.TempDir())
```

Comment:
Do you really need a whole function for this? I mean:

`fmt.Sprintf("%s/crash.zip", t.TempDir())`
vs `getTempDownloadPath(t)`

You're adding 3 lines of code to create this function but saving very little

Reply:
improved